README.md

```diff
@@ -436,4 +437,434 @@ To order them:
       -c <cohort id> \
    ```
 
-3. Update `_config.yml` file to add new cohort in collection
\ No newline at end of file
+3. Update `_config.yml` file to add new cohort in collection
+
+## License
+
+Attribution-ShareAlike 4.0 International
```

Comment:
We do not the full license here in the README file. Just a small bit like in the footer of the website

Reply:
I understand now 
i added a small bit but it was in the middle.
i will carry out changes now